docs/GettingStartedGuide.md

```diff
@@ -38,7 +53,7 @@ You can use the Data SDK in your CMake project or install it on your system.
 
 **To install the Data SDK on your system:**
 
-1. Install all the dependencies needed for the Data SDK.<br>For more information on dependencies, see the [Dependencies](#dependencies) and [Additional Linux dependencies](#additional-linux-dependencies) sections.
+1. Install all the dependencies needed for the Data SDK.<br>For more information on dependencies, see the [Dependencies](../README.md#dependencies) and [Additional Linux dependencies](../README.md#additional-linux-dependencies) sections in the README.md file.
```

Comment:
If you change the "Additional Linux dependencies" section according to the comment above, make sure to also update the link here.

Reply:
Changed.